tools/local-sensor/main.go

```diff
@@ -77,11 +74,25 @@ type localSensorConfig struct {
 	Delay              time.Duration
 }
 
+const (
+	jsonFormat string = "json"
+	rawFormat  string = "raw"
+)
+
+func isValidOutputFormat(format string) bool {
+	var validFormats = map[string]struct{}{
+		jsonFormat: {},
```

Comment:
A ridiculous nitpick: this map could hold the "formatter" function that receives the outfile path. You can still use it to check if the format is valid and use the static function to do the work at the end. I.e. remove the `switch` case.

Reply:
Done!

Reply:
Amazing. Thanks!